### PR TITLE
Using reactstrap for UI, basic cleanup

### PR DESCRIPTION
**Pull request by devinmatte**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react';
```
Comment (multiojuice) [inline src/App.jsx:1]: Why did you remove the spaces?

File: src/components/DBInfo/DBInfo.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 
 class DBInfo extends Component {
-  constructor(props) {
-    super(props);
-  }
+    constructor(props) {
+        super(props);
```
Comment (multiojuice) [inline src/components/DBInfo/DBInfo.jsx:5]: What standards for spacing are we using

File: src/App.jsx (hunk 1 above)
Comment (multiojuice) [reply src/App.jsx:1]: Are you using a linter?

File: src/components/DBInfo/DBInfo.jsx (hunk 2 above)
Comment (devinmatte) [reply src/components/DBInfo/DBInfo.jsx:5]: I've been using webstorm, so the defaults there is 4 space tabs

File: src/App.jsx (hunk 1 above)
Comment (devinmatte) [reply src/App.jsx:1]: Yes I am using a linter, I need to add the rc to the project oops
